htpasswd_migration.sh

```diff
@@ -86,4 +86,4 @@ case $var in
 		echo "Please enter correct number"
 		;;
 esac
-done
+done
```

Comment:
Revert. There must be a newline at the end of every file, unless there's a very good reason to the contrary.

Reply:
It's done.